src/parser.js

```diff
@@ -144,9 +144,13 @@ const parseLatex = (latex) => {
 const parseToken = (token, structure) => {
   const isFunction = functions.reduce((acc, val) => acc || val === token, false)
   if (isFunction) {
+    let tokenValue = token
```

Comment:
Can you please make the token equal 'int' instead of 'integral', so it matches the [ASCII Math](http://asciimath.org/) standards.

Just remove all changes in parser.js

So `\int \fraction{1}{x}`, becomes `int (1)/(x)`